EasyLearn/.gitignore

```diff
@@ -0,0 +1,37 @@
+# See https://help.github.com/articles/ignoring-files for more about ignoring files.
+#
+# If you find yourself ignoring temporary files generated by your text editor
+# or operating system, you probably want to add a global ignore instead:
+#   git config --global core.excludesfile '~/.gitignore_global'
+
+# Ignore bundler config.
+/.bundle
+
+# Ignore the default SQLite database.
+/db/*.sqlite3
+/db/*.sqlite3-*
+
+# Ignore all logfiles and tempfiles.
+/log/*
+/tmp/*
+!/log/.keep
+!/tmp/.keep
+
+# Ignore pidfiles, but keep the directory.
+/tmp/pids/*
+!/tmp/pids/
+!/tmp/pids/.keep
+
+# Ignore uploaded files in development.
+/storage/*
+!/storage/.keep
+/tmp/storage/*
+!/tmp/storage/
+!/tmp/storage/.keep
+
+/public/assets
+
+# Ignore master key for decrypting credentials and more.
+/config/master.key
+
+/config/database.yml
```

Comment:
Nice, but where is it your database.yml.example file?

Reply:
I added the database.yml.example in a new commit

Reply:
In the previous kata 'reloj checador', someone told us that we don't need include the database.yml, and told us that put it in the .gitignore file, for that reason we ignore the database.yml

Reply:
ooh sorry, I did'nt see the previous comment

Reply:
Yes @MauricioNC the database.yml is the file that contain the credentials of database and it's important that each one of your have a unique configuration on your local project, but into this repositorie we need to see a structure reference of this  file and it is the database.example.yml